### ISSUE TITLE
frameshift needs mafft

### ISSUE THREAD

**Issue by bioinformatikLabormedizinUSB** [cbg-ethz/smallgenomeutilities#17]
This is not mentioned in the dependencies.
Cheers,
P

**Comment by DrYak** [comment]
Yes, indeed. The current mecanism used in setup.py can only specify python libraries dependencies.

We would advise strongly to use the [bioconda packages](https://bioconda.github.io/recipes/smallgenomeutilities/README.html) whose format is able to handle non-python dependencies and can handle such installation requirements for you.

**Comment by DrYak** [comment]
I've slightly updated the README.md of version 0.3.9
The ["_Dependencies_" section](https://github.com/cbg-ethz/smallgenomeutilities#dependencies) makes clearer the dependence on mafft as an external tool , and the ["_Installation_" section](https://github.com/cbg-ethz/smallgenomeutilities#installation) lists bioconda as our prefered installation method.

**Comment by DrYak** [comment]
I still haven't [managed to gain access to the Pypi repository](https://github.com/pypa/pypi-support/issues/2340) though. So the version there is horribly outdated (it's still at 0.3).

**Comment by DrYak** [comment]
Update regarding the MAFFT dependency:
it is now possible to provide a [.chain file ](https://genome.ucsc.edu/goldenPath/help/chain.html) (e.g. produced with [bcftools' `--chain` option when calling the consensus](https://samtools.github.io/bcftools/bcftools.html#consensus)) that describes the mapping between the reference and consensus instead of re-aligning with mafft.
This dependency is therefor not manadatory anymore.

It is already in master and will be publishing in v0.4 in the comming days.

**Comment by DrYak** [comment]
Update:

smallgenomeutilities 0.4 released, available on [bioconda](https://anaconda.org/bioconda/smallgenomeutilities) (well, the [webpage](https://bioconda.github.io/recipes/smallgenomeutilities/README.html) hasn't updated yet, but the package is there) and on [PyPI](https://pypi.org/project/smallgenomeutilities/0.4.0/).

The [Galaxy Tool for running frameshift_deletions_check](https://usegalaxy.eu/root?tool_id=smgu_frameshift_deletions_checks) will be updated shortly.

**Comment by gordonkoehn** [comment]
Given this has been inactive for two years, I shall close it. It appears completed.
